File: src/main/java/org/jukeboxmc/event/player/PlayerToggleSprintEvent.java
package org.jukeboxmc.event.player;

import org.jukeboxmc.event.Cancelable;
import org.jukeboxmc.player.Player;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class PlayerToggleSprintEvent extends PlayerEvent implements Cancelable {

    private boolean isSprinting;

    public PlayerToggleSprintEvent(Player player, boolean isSprinting) {
        super(player);
        this.isSprinting = isSprinting;
    }

    public boolean isSprinting() {
        return this.isSprinting;
    }
}

File: src/main/java/org/jukeboxmc/event/player/PlayerToggleSprintEvent.kt
package org.jukeboxmc.event.player

import org.jukeboxmc.event.Cancellable
import org.jukeboxmc.player.Player

/**
 * @author LucGamesYT
 * @version 1.0
 */
class PlayerToggleSprintEvent
/**
 * Creates a new [PlayerToggleSprintEvent]
 *
 * @param player      who toggled sprinting
 * @param isSprinting whether the player is sprinting
 */(
    player: Player,
    /**
     * Retrieves whether the player is sprinting or not
     *
     * @return whether the player is sprinting
     */
    val isSprinting: Boolean
) : PlayerEvent(player), Cancellable